rpcserver/router.go

```diff
@@ -677,6 +677,10 @@ func (server *routedBoltzServer) createSwap(ctx context.Context, isAuto bool, re
 		if err != nil {
 			return nil, err
 		}
+		if decoded.Expiry().Minutes() < boltz.GetBlockTime(pair.From) {
```

Comment:
Does `Expiry()` return the expiry value in the invoice or *time until* expiry? And how about some buffer?

Reply:
The time until expiry:
```
	// expiry specifies the timespan this invoice will be valid.
	// Optional. If not set, a default expiry of 60 min will be implied.
```
Lets go for a 10s buffer